mm2src/mm2_main/tests/mm2_tests/z_coin_tests.rs

```diff
@@ -47,7 +47,7 @@ async fn withdraw(mm: &MarketMakerIt, coin: &str, to: &str, amount: &str) -> Tra
 }
 
 #[test]
-pub fn activate_z_coin_light() {
+pub(crate) fn activate_z_coin_light() {
```

Comment:
I still don't understand why a test function has to be `pub(crate)`

Reply:
oops, this update is meant for https://github.com/KomodoPlatform/komodo-defi-framework/blob/d775aec10469946a4d49c50c40342bc4dbcc8636/mm2src/mm2_test_helpers/src/for_tests.rs#L3101

Reply:
wrong place.. updated